lib/sycamore/sycamore/llms/bedrock.py

```diff
@@ -56,10 +60,20 @@ def __init__(
         self._client = boto3.client(service_name="bedrock-runtime")
         super().__init__(self.model.name, cache)
 
+    def __reduce__(self):
+        def deserializer(kwargs):
+            return Bedrock(**kwargs)
+
+        kwargs = {"model_name": self.model_name, "cache": self._cache}
+        return deserializer, (kwargs,)
+
     def is_chat_mode(self) -> bool:
         """Returns True if the LLM is in chat mode, False otherwise."""
         return True
 
+    def format_image(self, image: Image.Image) -> dict[str, Any]:
```

Comment:
Is this true for all Bedrock models or only Anthropic? If you can only support anthropic models here just throw an error if the model_name does not indicate an anthropic model.

Reply:
Good call. Will do.